RevenueCat/Plugins/iOS/PurchasesUnityHelper.m

```diff
@@ -203,6 +208,36 @@ - (void)checkTrialOrIntroductoryPriceEligibility:(NSArray *)productIdentifiers
     }];
 }
 
+- (void)invalidatePurchaserInfoCache
+{
```

Comment:
there's some inconsistency in how we're doing formatting of `{`. 
In some methods, we're doing it in the line _after_ the signature, and in others we're doing it on the same line. 
same line seems to be the most prevalent in purchases-ios, so I'd go with that. 

No need to update the entire codebase for it, but we should pick a standard and apply it consistently from now on.